### DISCUSSION TITLE
Error whitelist

### DISCUSSION THREAD

**Discussion by projetAziz** [Web3Auth/Web3Auth#657] [Help]
hi , can anyone help me to solve this error.
i deployed my next js app using web3Auth, but i got this error 
could not validate redirect, please whitelist your domain: http://......:3000 for provided clientId "mycliantId" at https://dashboard.web3auth.io

**Comment by himanshuchawla009** [comment]
https://web3auth.io/docs/developer-dashboard/get-client-id

Refer to this doc
